### DISCUSSION TITLE
Medium range radio - CC1101 or other TI versus Si443x or other SiLabs radio

### DISCUSSION THREAD

**Discussion by HeadBoffin** [jgromes/RadioLib#959] [Q&A]
Mostly I use LoRa but I have an idea for domestic garden monitoring that doesn't need the range that LoRa provides, 1,000m maximum would be sufficient and would need to be a more affordable, typical price for a LoRa radio is > $6, so less than that.

Both TI & SiLabs provide a blizzard of information, so much so that I can hear the bell toll but don't know which clanger it is (aka I can no longer see the wood for the trees).

Does anyone have experience of both that can make a recommendation. Or if you've only used one or the other, comment on their experience.

Much appreciated, Nick

**Comment by jgromes** [comment]
I can't really say from experience about either CC1101 or Si443x, since lately it's been mostly just LoRa vs anything else sub-GHz, at least for me. Though I honestly don't see any "killer feature" in either TI or SiLabs designs, maybe it does exist in your use case (e.g. slightly lower power consumption).

I don't know what volume you're going for, but it seems to me like the price difference of LoRa modules vs other sub-GHz transceivers is not going to be that significant unless we are talking about 100s of units. Especially if you would be fine with using the older SX127x - you can get those for $4-5 off Ebay.

Also, I wanted to point out the FDRS project (even though you specifically asked about non-LoRa), perhaps it could be useful: https://github.com/timmbogner/Farm-Data-Relay-System

**Comment by HeadBoffin** [comment]
eBay is fine for personal projects but mostly I buy stuff from Mouser / DigiKey and some of the ISM sub-GHz modules are half the price or less than a LoRa implementation. I need to be mindful of power consumption so I can keep the power unit (likely to be solar) small so that's a factor too. But mostly just spitballing on the basis that there are drivers here for the TI & SiLabs offerings.
